src/libs/ajax.js

```diff
@@ -66,8 +74,8 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
       // requesterPaysError is true if the request requires a user project for billing the request to. Such errors
       // are not transient and the request should not be retried.
       const shouldRetry = !error.requesterPaysError
-
-      if (!shouldRetry) {
+      if (shouldRetry) {
```

Comment:
This reverses the retry logic and would only retry requester pays errors.

Reply:
> Confused was by the use of double `!`. Unsure why use a double not-not?

I suppose this could have been written as:
```js
const shouldNotRetry = Boolean(error.requesterPaysError)

if (shouldNotRetry) {
  throw error
}
```

However, negative variables are generally difficult to work with, since it's very easy to end up in a situation with a double negative that is (as you point out) difficult to understand (for example, `if (!shouldNotRetry) { ...`.